l5kit/l5kit/sampling/agent_sampling.py

```diff
@@ -149,10 +142,13 @@ def generate_agent_sample(
         "history_positions": history_coords_offset,
         "history_yaws": history_yaws_offset,
         "history_availabilities": history_availability,
-        "world_to_image": world_to_image_space,
-        "centroid": agent_centroid,
-        "yaw": agent_yaw,
-        "extent": agent_extent,
+        "world_to_image": raster_from_world,  # TODO deprecate
+        "raster_from_agent": raster_from_world @ np.linalg.inv(agent_from_world),
+        "raster_from_world": raster_from_world,
+        "agent_from_world": agent_from_world,
+        "centroid": agent_centroid_m,
+        "yaw": agent_yaw_rad,
+        "extent": agent_extent_m,
```

Comment:
After the challenge we should consider change key to `agent_position_m`, `agent_yaw_rad`, `agent_extent_m`